packages/fast-components-react-base/src/button/button.spec.tsx

```diff
@@ -0,0 +1,95 @@
+import * as React from "react";
+import * as ShallowRenderer from "react-test-renderer/shallow";
+import * as Adapter from "enzyme-adapter-react-16";
+import { configure, shallow } from "enzyme";
+import examples from "./examples.data";
+import { generateSnapshots } from "@microsoft/fast-jest-snapshots-react";
+import {
+    ButtonHTMLTags,
+    ButtonProps,
+    IButtonClassNameContract,
+    IButtonHandledProps,
+    IButtonManagedClasses,
+    IButtonUnhandledProps
+} from "./button";
+
+/*
+ * Configure Enzyme
+ */
+configure({adapter: new Adapter()});
+
+describe("button", (): void => {
+    generateSnapshots(examples);
+});
+
+describe("button unit-tests", (): void => {
+    const Component: React.ComponentClass<IButtonHandledProps & IButtonManagedClasses> = examples.component;
+    const managedClasses: IButtonClassNameContract = {
+        button: "test-button"
+    };
+    const href: string = "https://www.microsoft.com";
+
+    test("should return an object that includes all valid props which are note enumarated as handledProps", () => {
+        const handledProps: IButtonHandledProps & IButtonManagedClasses = {
+            managedClasses
+        };
+
+        const unhandledProps: IButtonUnhandledProps = {
+            "aria-hidden": true
+        };
+
+        const props: ButtonProps = {...handledProps, ...unhandledProps};
+
+        const rendered: any = shallow(

```

Comment:
can we type these more precisely than "any"?

Reply:
Tried importing the typings and they aren't taking...leaving this for now.